---
date: "2022-02-06"
external_link: ""
authors: ["cdrl"]
image:
  caption: 
  focal_point: Smart
links:
- icon: link
  icon_pack: fas
  name: Link
  url: http://www.ilincs.org/ilincs/
- icon: file-alt
  icon_pack: fas
  name: Paper
  url: https://www.biorxiv.org/content/10.1101/826271v2

title: iLINCS
summary: iLINCS (Integrative LINCS) is an integrative web platform for analysis of LINCS data and signatures
tags:
- Resources
- Protein
- Drug
- Gene
- Database
- Network
- Interactions


---


iLINCS (Integrative LINCS) is an integrative web platform for analysis of LINCS data and signatures. The portal provides biologists-friendly user interfaces for analyzing transcriptomics and proteomics LINCS datasets. The portal integrates R analytical engine via several R tools for web-computing (rserve, opencpu, Shiny, rgl) and DCIC developed web tools and applications (FTreeView, Enrichr) into a coherent web platform for LINCS data analysis. Users can follow several workflows which allow them to identify differentially expressed genes, proteins and phosphoproteins in LINCS datasets and use them in analysis of other LINCS and non-LINCS dataset (eg TCGA and GEO transcriptomic datasets), and in the analysis of LINCS L1000 signatures. In this way, the platform facilitates integrative analysis of LINCS data and signatures. The mechanistic interpretation of LINCS transcriptomic and proteomics signatures is facilitated by the enrichment analysis via Enrichr and DAVID, and by pathways analysis using the R implementation of the SPIA algorithm. The portal can be accessed freely and does not require user registration. 

Scientists are able to compare a "disease transcriptional signature", a gene expression profile, to a library of "drug activity transcriptional signatures" to generate a Connectivity Map based on Connectivity score. Positive correlation between drug perturbations and disease gene expression profiles suggest common gene expression changes, while negative correlation is suggestive of drug perturbation ability to "reverse" the disease.

Portal also provides tools to analyze list of differentially expressed genes in other LINCS and non-LINCS datasets (i.e. TCGA, GEO, etc.) with an option to find concordant LINCS and non-LINCS gene expression signatures.




