---
date: "2020-10-24T00:00:00Z"
external_link: ""
authors: ["khaled"]
image:
  caption: 
  focal_point: Smart
links:
- icon: file-alt
  icon_pack: fas
  name: Paper
  url: https://pubmed.ncbi.nlm.nih.gov/34919543/
- icon: solar-panel
  icon_pack: fas
  name: Package Website
  url: https://cogdisreslab.github.io/KRSA/
- icon: github
  icon_pack: fab
  name: GitHub
  url: https://github.com/CogDisResLab/KRSA 
- icon: link
  icon_pack: fas
  name: Shiny app
  url: https://kalganem.shinyapps.io/KRSA/

title: KRSA App
summary: KRSA is an R package and R Shiny web application for an end-to-end upstream kinase analysis of kinome array data
tags:
- Apps
- Projects
- R Shiny
- Bioinformatics
- Web Application

url_code: ""
url_pdf: ""
url_slides: ""
url_video: ""

featured: true

---


Phosphorylation by serine-threonine and tyrosine kinases is critical for determining protein function. Array-based platforms for measuring reporter peptide signal levels allow for differential phosphorylation analysis between conditions for distinct active kinases. Peptide array technologies like the PamStation12 from PamGene allow for generating high-throughput, multi-dimensional, and complex functional proteomics data. As the adoption rate of such technologies increases, there is an imperative need for software tools that streamline the process of analyzing such data. 

We present Kinome Random Sampling Analyzer (**KRSA**), an R package and R Shiny web-application for analyzing kinome array data to help users better understand the patterns of functional proteomics in complex biological systems. KRSA is an All-In-One tool that reads, formats, fits models, analyzes, and visualizes PamStation12 kinome data.

More info on the PamStation12 platform can be found here: [PamGene](https://pamgene.com/)







